test/integration/standard_exit_test.exs

```diff
@@ -0,0 +1,98 @@
+defmodule Integration.StandardExitTest do
+  @moduledoc """
+  Integration tests for the full standard exit flows.
+
+  This is running against no childchain/operator or watcher. Instead it mocks
+  out how we need to extract the data, specifically utxo tracking, to create
+  the correct exit data that the contracts can use to start an standard exit.
+  """
+
+  use ExUnit.Case
+
+  alias ExPlasma.Block
+  alias ExPlasma.Client
+  alias ExPlasma.Transaction
+  alias ExPlasma.Transactions.Deposit
+  alias ExPlasma.Transactions.Payment
+  alias ExPlasma.Utxo
+
+  import ExPlasma.Client.Config,
+    only: [authority_address: 0]
+
+  @moduletag :integration
+
+  setup do
+    # On a fresh contract deploy, we need to invoke the contract to add
+    # the exit queue you want explicitly.
+    Client.add_exit_queue(1, <<0::160>>)
+    :ok
+  end
+
+  test "should be able to standard exit on a valid transaction" do
+    # Generate a deposit and return the usuable utxo
+    utxo = deposit(from: authority_address(), amount: 1)
+
+    # Send a transaction and create the exit data we need from it.
+    %{tx_bytes: tx_bytes, utxo_pos: utxo_pos, proof: proof} =
+      transact(to: authority_address(), utxo: utxo)
+
+    assert {:ok, _receipt_hash} =
+             Client.start_standard_exit(tx_bytes, %{
```

Comment:
this actually does not belong in the "production code" if your intention is to include ex_plasma in `elixir-omg`. starting a standard exit is not part of any API exposed by watcher or childchain. At this point I'm not exactly sure what the purpose of this test is - the interaction itself belongs to plasma-contracts.